### PR TITLE
[GoLint] Fixed golint errors when syncing to Google3.

### PR DESCRIPTION
**Pull request by mingzhi**

### CODE REVIEW

**Comment by mingzhi** [comment]
Thank you Rob!!!

File: rules/aip0131/message_rules_test.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ import (
 
 	"github.com/jhump/protoreflect/desc"
 	"github.com/jhump/protoreflect/desc/builder"
-	"google.golang.org/genproto/protobuf/field_mask"
+	fpb "google.golang.org/genproto/protobuf/field_mask"
```
Comment (lukesneeringer) [inline rules/aip0131/message_rules_test.go]: Why were these lint errors exactly?

**Comment by lukesneeringer** [comment]
Hey, can we talk about this one?
If GitHub is our source of truth, we can not expect (potentially external) rule authors to conform to arbitrary lint rules that are not enforced _here_. This is going to be a consistent problem.

I would argue that:
(1) We should probably enforce linting here (as we already do).
(2) We should probably disable any linting in google3. It should accept whatever code is accepted here, with only the modifications required to get it _running_ in google3.

File: rules/aip0131/message_rules_test.go (hunk 1 above)
Comment (mingzhi) [reply rules/aip0131/message_rules_test.go]: Protocol buffer packages should have short names that end in "pb" to distinguish with other canonical Go packages.

**Pull request merged**